requirements-dev.txt

```diff
@@ -48,38 +43,27 @@ mypy==0.971
 mypy-extensions==0.4.3
 myst-parser==0.18.1
 nodeenv==1.7.0 ; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4, 3.5, 3.6'
-packaging==21.3 ; python_version >= '3.6'
+packaging==23.0 ; python_version >= '3.7'
```

Comment:
was this bump intentional?

Reply:
When you run `make requirements`, it will make all packages pinned in the `requirements-*.txt`
The requirements were outdated from `Pipenv.lock`

Reply:
- https://github.com/vmware/repository-service-tuf-cli/issues/184